### ISSUE TITLE
Mountebank UI Log doesnt print XML requests properly

### ISSUE THREAD

**Issue by vinitmalhotra** [bbyars/mountebank#376]
Hi All,
I have written a script in which i am logging the incoming request xml using logger.info(). I am able to see the request xml properly on the terminal in my local system but on the MB UI it doesnt print the XML tags.

#### Expected behaviour

Mountebank UI running on localhost:2525 should display the XML request properly.

<StatusCode>0</StatusCode>
        <ServerStatuscode>Success</ServerStatuscode>
        <Severity>Info</Severity>
        <StatusDesc>Successful</StatusDesc>

#### Actual behaviour

It doesn't show the XML tags. It displays the below :-

0
       Success
      Info
 Successful

Basically on the MB UI - it only displays the JSON but not the XML.
#### Software versions used

```
OS         : Mac
mountebank :
node.js    :
  (only if installed via npm)
Installation method :
  (npm, zip, tar, pkg, deb, rpm)

This issue was raised on the google groups, has been accepted by Brandon and we are waiting for the fix to be delivered to us. Can someone look into it on priority.

https://groups.google.com/forum/#!topic/mountebank-discuss/pkWJPFwXV3I
